src/e2e-test/features/gcs/source/GCSToGCSAdditonalTests.feature

```diff
@@ -85,3 +85,306 @@ Feature: GCS source - Verification of GCS to GCS Additional Tests successful
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Validate the data from GCS Source to GCS Sink with expected csv file and target data in GCS bucket
+
+  @GCS_CSV @GCS_SINK_TEST
+  Scenario: To verify data is getting transferred from GCS Source to GCS Sink using test Schema Detection On Single File without connection
+    Given Open Datafusion Project to configure pipeline
+    When Select plugin: "GCS" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "GCS" from the plugins list as: "Sink"
+    Then Connect plugins: "GCS" and "GCS2" to establish connection
+    Then Navigate to the properties page of plugin: "GCS"
+    Then Select dropdown plugin property: "select-schema-actions-dropdown" with option value: "clear"
+    Then Replace input plugin property: "project" with value: "projectId"
+    Then Override Service account details if set in environment variables
+    Then Enter input plugin property: "referenceName" with value: "sourceRef"
+    Then Enter GCS source property path "gcsCsvDataFile"
+    Then Select GCS property format "delimited"
+    Then Enter input plugin property: "delimiter" with value: "delimiterValue"
+    Then Toggle GCS source property skip header to true
+    Then Validate "GCS" plugin properties
+    Then Verify the Output Schema matches the Expected Schema: "gcsSingleFileDataSchema"
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "GCS2"
+    Then Replace input plugin property: "project" with value: "projectId"
+    Then Override Service account details if set in environment variables
+    Then Enter input plugin property: "referenceName" with value: "sourceRef"
+    Then Enter GCS sink property path
+    Then Select dropdown plugin property: "select-format" with option value: "csv"
+    Then Validate "GCS" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Click on preview data for GCS sink
+    Then Verify preview output schema matches the outputSchema captured in properties
+    Then Close the preview data
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    Then Validate the data from GCS Source to GCS Sink with expected csv file and target data in GCS bucket
+
+  @GCS_CSV @GCS_SINK_TEST
+  Scenario: To verify the pipeline is getting failed from GCS to GCS when Schema is not cleared in GCS source On Single File
```

Comment:
nit: `To verify the pipeline....when `default schema` is not cleared....`

Reply:
Changed it to **To verify the pipeline is getting failed from GCS to GCS when default schema is not cleared in GCS source On Single File**